src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -1035,15 +1035,33 @@ public Response toggleSuperuser(@PathParam("identifier") String identifier) {
 		ActionLogRecord alr = new ActionLogRecord(ActionLogRecord.ActionType.Admin, "toggleSuperuser")
```

Comment:
I'm just leaving this note at the top but can you please add a release note snippet? Please see https://preview.guides.gdcc.io/en/develop/developers/version-control.html#writing-release-note-snippets

Reply:
@pdurbin  done